### PR TITLE
Peer code review PR

### PR DESCRIPTION
**Pull request by becks135**
thanks for reviewing!

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,21 +1,130 @@
...
+        const itemCount = shoppingCart[item].numberInCart;
+        cartCounter += itemCount;
+      }
+      setNumOfItemsInCart(cartCounter);
+    })
+  }
+
+  function getFilteredResults(categoryName){
+    const filteredResult = inventory.filter(item=>item.category===categoryName);
+    // console.log(inventory);
```
Comment (89rp) [inline src/App.js:48]: make sure to remove console logs 😄

File: src/Components/ShoppingCartItem.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+        get(shoppingCartItemRef)
+            .then((snapshot) => {
+                //check number of items in shopping cart and add back to inventory
+                const numberToAddToInventory = snapshot.val().numberInCart;
+                addItemsToInventory(item.id,numberToAddToInventory);
+                //remove item from shopping cart database
+                remove(shoppingCartItemRef);
+            })
+            .catch((err) =>
+                //TODO: update error message
```
Comment (89rp) [inline src/Components/ShoppingCartItem.js:28]: To do reminder ✔️

File: src/modules/manageInventory.js [hunk 3]
```diff
@@ -0,0 +1,54 @@
+//Modules
+import { getDatabase, ref, get, onValue, update } from "firebase/database";
+
+//config
+import firebase from "../firebase";
+
+const database = getDatabase(firebase);
+const inventoryRef = ref(database,"story-inventory");
+//TODO:correct spelling error - story-inventory
```
Comment (89rp) [inline src/modules/manageInventory.js:9]: To do reminder ✔️

File: src/App.js [hunk 4]
```diff
@@ -1,21 +1,130 @@
...
+    return filteredResult;
+  }
+
+
+  useEffect(() => {
+    getInventoryFromDB();
+
+    // TODO: move this to another useEffect??
```
Comment (89rp) [inline src/App.js:56]: To do reminder ✔️

File: src/App.js [hunk 5]
```diff
@@ -1,21 +1,130 @@
...
+    getNumOfItemsInCart();
+    // setFilteredInventory(getFilteredResults("electronics"));
+  }, []);
+
   return (
+    // TODO: add credit to fakedataAPI
```
Comment (89rp) [inline src/App.js:62]: To do reminder ✔️

File: src/App.js [hunk 6]
```diff
@@ -1,21 +1,130 @@
...
+    // TODO add back to top button
```
Comment (89rp) [inline src/App.js:63]: To do reminder ✔️

File: src/Components/Item.js [hunk 7]
```diff
@@ -1,20 +1,68 @@
...
+        .then((snapshot) => {
+            //if item already exists in the shopping cart, add one to the number of items
+            if (snapshot.exists()) {
+                const numberInCart = snapshot.val().numberInCart;
+                update(shoppingCartItemRef, {"numberInCart": numberInCart + 1 });
+            }
+            //if item does not exist, add item object to the database
+            else {
+                //create object to add to shopping cart database
+                //TODO: move this to it's own function
```
Comment (89rp) [inline src/Components/Item.js:30]: To do reminder ✔️

File: src/modules/manageInventory.js (hunk 3 above)
Comment (becks135) [reply src/modules/manageInventory.js:9]: Thanks for the reminder. Will update the spelling error in the database after project submission

File: src/firebase.js [hunk 8]
```diff
@@ -0,0 +1,20 @@
+// Import the functions you need from the SDKs you need
+import { initializeApp } from "firebase/app";
+// TODO: Add SDKs for Firebase products that you want to use
```
[the remark this answers is not in the document]
Comment (becks135) [reply src/firebase.js:3]: default comment in firebase config file.

File: src/Components/Item.js (hunk 7 above)
Comment (becks135) [reply src/Components/Item.js:29]: thanks Rakesh! I will revisit after project submission

**Pull request merged**